cmd/opm/internal/util/util.go

```diff
@@ -66,3 +67,11 @@ func CreateCLIRegistry(cmd *cobra.Command) (*containerdregistry.Registry, error)
 	}
 	return reg, nil
 }
+
+func OpenFileOrStdin(cmd *cobra.Command, args []string) (io.ReadCloser, string, error) {
```

Comment:
Nit: This is an internal function, and I know it was mostly a move, but the implementation only ever looks at the first argument. It seems reasonable to just make this a single string argument instead of a slice of strings
```suggestion
func OpenFileOrStdin(cmd *cobra.Command, path string) (io.ReadCloser, string, error) {
```

Reply:
This has bugged me too, but it does actually check for [empty args](https://github.com/operator-framework/operator-registry/pull/1335/files#diff-9373b2d6ffc7be864852e489772ff2062a0a2359ff9f3be9f574ba26d69b8fedR72) as a condition for reading from stdin instead.
We could make it pass just the input path and have each caller test for empty args (and pass in an empty string) but then the functions utility is significantly reduced.

Reply:
I don't feel super strongly one way or the other, and saw the empty args check but figured an empty string would work equivalently. This shouldn't block this PR if you'd rather keep it as is.